internal/deb/extract.go

```diff
@@ -34,16 +34,26 @@ type ExtractInfo struct {
 	Optional bool
 }
 
-func checkExtractOptions(options *ExtractOptions) error {
+func validateExtractOptions(options *ExtractOptions) (*ExtractOptions, error) {
```

Comment:
silly-silly-nit: `validate` means to check, how about `reviseExtractOptions` or `amendExtractOptions` since we are adding stuff if the `Create` function is missing?

Reply:
For `validate` I have checked the dictionary and there are two meanings:
* demonstrate or support the truth or value of.
* make or declare legally valid.
I was using it because it combines both.

What about `getValidExtractOptions`? I like `reviseExtractOptions`  but revise is not a common word in the "coding world".

Reply:
`getValidExtractOptions` is cool with me. Upon second look, `validateExtractOptions` is fine with me too. It was a very silly nitpicking anyway.